cves/kernel/CVE-2020-12654.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  Method is called without first making a size check. This could cause a buffer
```

Comment:
Maybe add a little background knowledge of the CVE for better reader understanding. e.g. the name subsystem and what its responsible for, and or, what the specific method is responsible for.